### PR TITLE
Aftershock: Adjust rat snatcher stats

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Mods "Aftershock: Adjust rat snatcher stats"

#### Purpose of change
This monsters where exceedingly deadly for something that is supposed to control pests. And the players arent rats, usually, so they shouldn't be afraid of these.

#### Describe the solution
Stat adjustment to health, speed and attack damage to make the snatcher not overtly threatening 

#### Testing
Load game, test the bots cant deal more than scratch damage to a player.

#### Additional context
 They can still kill rats.

### CODE REVIEW

File: data/mods/Aftershock/items/inactiverobot.json [hunk 1]
```diff
@@ -430,15 +430,20 @@
...
-    "flags": [ "LEAK_DAM", "RADIOACTIVE", "TRADER_AVOID" ]
+    "flags": [ "TRADER_AVOID" ]
   },
   {
     "id": "bot_skitterbot_rat",
     "type": "TOOL",
     "copy-from": "bot_skitterbot_base",
     "name": "inactive rat snatcher",
-    "description": "A salvaged skitterbot repurposed for hunting small game.  It attacks targets with pincers and an integrated tazer.  Activate this item to deploy the robot.",
+    "description": "A small hexapod drone designed to hold small pests, folded into a small cube for easy of storage. Its capable of  hunting all sorts of small mammals with remarkable efficiency,  but shouldn't be expected to handle anything much bigger than a that.",
```
Comment (Maleclypse) [inline data/mods/Aftershock/items/inactiverobot.json:440]:
```suggestion
    "description": "A small hexapod drone designed to hold small pests, folded into a small cube for easy of storage. Its capable of  hunting all sorts of small mammals with remarkable efficiency,  but shouldn't be expected to handle anything much bigger than a rat.",
```

Comment (Maleclypse) [reply data/mods/Aftershock/items/inactiverobot.json:440]: Could also go cat or some other small mammal.

File: data/mods/Aftershock/items/inactiverobot.json [hunk 2]
```diff
@@ -430,15 +430,20 @@
     "bashing": 6,
     "symbol": ";",
     "color": "dark_gray",
-    "flags": [ "LEAK_DAM", "RADIOACTIVE", "TRADER_AVOID" ]
+    "flags": [ "TRADER_AVOID" ]
```
Comment (Maleclypse) [inline data/mods/Aftershock/items/inactiverobot.json:433]: Does trader avoid mean the trader won't buy or sell it or just that they won't buy it?

Comment (John-Candlebury) [reply data/mods/Aftershock/items/inactiverobot.json:433]: Prevents npcs with random inventories from starting witht them.

File: data/mods/Aftershock/mobs/robots.json [hunk 3]
```diff
@@ -377,20 +377,22 @@
     "type": "MONSTER",
     "copy-from": "mon_skitterbot",
     "name": "rat snatcher",
```
[the remark this answers is not in the document]
Comment (Maleclypse) [reply data/mods/Aftershock/mobs/robots.json:379]: “Colonial Pest Removal aka Rat Snatchers” something about that name doesn’t feel right but spitballing it to help flush out potential names.

[the remark this answers is not in the document]
Comment (John-Candlebury) [reply data/mods/Aftershock/mobs/robots.json:379]: Imo theres absolutely no need, rat snatcher is a perfectly fine name. Its short and descriptive, and tells you exactly how dangerous the robot is.

I think brand names are better kept for actually dangerous or particularily unique things.

**Review by Maleclypse — APPROVED** [review]

File: data/mods/Aftershock/mobs/robots.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mom-Bun) [reply data/mods/Aftershock/mobs/robots.json:379]: I can agree with that!

**Review by Mom-Bun — APPROVED** [review]

**Pull request merged**